libpod/container_inspect.go

```diff
@@ -831,11 +828,12 @@ func (c *Container) generateInspectContainerHostConfig(ctrSpec *spec.Spec, named
 	usernsMode := ""
 	if c.config.UserNsCtr != "" {
 		usernsMode = fmt.Sprintf("container:%s", c.config.UserNsCtr)
-	} else {
+	} else if ctrSpec.Linux != nil {
```

Comment:
Are these if's really coverity issues?

Reply:
Yes coverity was complaining about the chance the ctrSpec.Linux could be nil and dereferenced later.